addon/components/paper-item.js

```diff
@@ -19,9 +19,9 @@ export default Ember.Component.extend(RippleMixin, ProxyMixin, {
 
   hasProxiedComponent: Ember.computed.bool('proxiedComponents.length'),
 
-  hasSecondaryAction: Ember.computed('secondaryItem', 'action', function() {
+  hasSecondaryAction: Ember.computed('secondaryItem', 'onClick', function() {
```

Comment:
@xomaczar - Absolutely. The purpose of this PR was to make some API changes. The affected file has not been gone thru and adjusted to conform to the ember-paper style guides (in contributing.md). I only touch paper-item to get rid of all the "action" properties.